src/build_info.cpp

```diff
@@ -13,7 +13,7 @@ std::string BuildInfo::report()
       << "  LLVM: " << LLVM_VERSION_MAJOR << "." << LLVM_VERSION_MINOR << "."
       << LLVM_VERSION_PATCH << std::endl
       << "  unsafe uprobe: "
```

Comment:
Also please change this to:
```suggestion
      << "  unsafe probes: "
```